samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>In Hosted Pipelines beta and deployments with workload identity we won't have <code>user-gcp-sa</code>. Maybe leave a comment here to prevent confusion?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Do we need the tasks to run sequentially? Maybe we can get rid of ".after" and run them simultaneously?</p>

Reply:
good point ... removing the after.

Reply:
@numerology  updated with following comment LMK if this works. 
'    # This pipeline assumes a user-gcp-sa is needed for execution, if no secret is needed,
    # or a different secret is being used following should be updated accordingly. '